.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "lib/forge-std"]
```

Comment:
add submodule init instructions to readme. add submodule init to `yarn bootstrap` script at repo root